cbf2nexus -c zlib --list -o i03-1.h5 "${CBF_DATA_INPUT}/thaumatin_die_M1S5_1_0005_2.cbf"
--- a/src/cbf_simple.c
+++ b/src/cbf_simple.c
@@ -669,7 +669,7 @@ extern "C" {
                     
                     match = 0;
                     
-                    for(crow = 0; crow < count+1; crow++) {
+                    for(crow = 0; crow < count; crow++) {
                         
                         if (!cbf_cistrcmp(xscan_id,scanids[crow])) {
                             
